tests/metrics/test_pcr_metrics.py

```diff
@@ -0,0 +1,38 @@
+from tests.common import *
+import pandas as pd
+
+
+def test_pc_regression(adata):
+    scIB.me.pc_regression(adata.X, adata.obs["batch"])
```

Comment:
What is tested here?

Reply:
implementation... if it doesn't run, there's an errorr